StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -94,6 +96,10 @@ extension PaymentSheet {
                 return "Zip"
             } else if case .dynamic("amazon_pay") = self {
                 return "Amazon Pay"
+            } else if case .dynamic("alipay") = self {
+                return "Alipay"
+            } else if case .dynamic("oxxo") = self {
+                return "OXXO"
```

Comment:
aren't these already defined by line 90?

Reply:
Ah! didn't notice that, I was just following copy pasting